src/JSInterop/Mono.WebAssembly.Interop/src/MonoWebAssemblyJSRuntime.cs

```diff
@@ -16,6 +16,25 @@ namespace Mono.WebAssembly.Interop
     /// </summary>
     public class MonoWebAssemblyJSRuntime : JSInProcessRuntime
     {
+        /// <summary>
+        /// Gets the <see cref="MonoWebAssemblyJSRuntime"/> used to perform operations using <see cref="DotNetDispatcher"/>.
+        /// </summary>
+        private static MonoWebAssemblyJSRuntime Instance { get; set; }
+
+        /// <summary>
+        /// Initializes the <see cref="MonoWebAssemblyJSRuntime"/> to be used to perform operations using <see cref="DotNetDispatcher"/>.
+        /// </summary>
+        /// <param name="jsRuntime">The <see cref="MonoWebAssemblyJSRuntime"/> instance.</param>
+        protected static void Initialize(MonoWebAssemblyJSRuntime jsRuntime)
```

Comment:
We want this extensibility to configure the `ElementRefJsonConverter` here: https://github.com/aspnet/AspNetCore/pull/13154/files#diff-d74d8a63556f3a7f0b5be4814c52aeccR19, but it has to be static. I'm open to suggestions. We could also revisit this post 3.0 since this is client-side blazor and we have some additional time to figure this out.

Reply:
Why does it need to be static? 

So the problem here is you want this class to be subclassable, but it also has a static singleton property. The only winning move is not to play.

If you don't have a static property here, then this becomes an instance method, and it's wicked simple.